rust_snuba/src/strategies/clickhouse.rs

```diff
@@ -73,6 +73,10 @@ impl TaskRunner<BytesInsertBatch, BytesInsertBatch, anyhow::Error> for Clickhous
                 timer!("insertions.batch_write_ms", elapsed);
             }
             counter!("insertions.batch_write_msgs", insert_batch.len() as i64);
+            gauge!(
```

Comment:
why a gauge? if there are multiple batches written within the same second, you won't be able to read min/max/avg, no?

Reply:
ok changed it to a counter